File: WildfireSimulation/Models/WeatherAudit.cs
namespace WildfireSimulation.Models
{
    public class WeatherAudit
    {
        public List<LightningEvent>? lightningEvents {  get; set; } = new List<LightningEvent>();
        public List<WindEvent>? WindEvents { get; set; } = new List<WindEvent>();
        public List<RainEvent>? RainEvents { get; set; } = new List<RainEvent>();
    }
}

File: WildFire.Web/src/app/models/WeatherAudit.ts
// import { LightningEvent } from "./LightningEvent";
// import { RainEvent } from "./RainEvent";
// import { WindEvent } from "./WindEvent";

// export class WeatherAudit{
//     lightningEvents?: Array<LightningEvent> = [];
//     windEvents?: Array<WindEvent> = [];
//     rainEvents?: Array<RainEvent> = [];

//     // constructor(light: Array<LightningEvent>, wind: Array<WindEvent>, rain: Array<RainEvent>){

//     //     this.lightningEvents = light;
//     //     this.windEvents = wind;
//     //     this.rainEvents = rain;
//     // }
//     constructor(){}
// }

import { LightningEvent } from "./LightningEvent";
import { RainEvent } from "./RainEvent";
import { WindEvent } from "./WindEvent";

export interface WeatherAudit{
    lightningEvent?: Array<LightningEvent>;
    windEvents?: Array<WindEvent>;
    rainEvents?: Array<RainEvent>;

    // constructor(light: Array<LightningEvent>, wind: Array<WindEvent>, rain: Array<RainEvent>){

    //     this.lightningEvents = light;
    //     this.windEvents = wind;
    //     this.rainEvents = rain;
    // }
}